packages/relic/lib/src/method/method.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:relic/src/headers/typed/typed_headers.dart';
+
+/// Represents the HTTP methods used in requests as constants.
+class Method extends TypedHeader {
```

Comment:
Suggestion: This isn't a header.

It is simply a value that is used in header.
It shouldn't extend the TypedHeader interface right?

Reply:
Yes it is, the `Request Method` is a header.